src/Loader.php

```diff
@@ -72,7 +72,10 @@ public function __construct(ClassLoader $loader, $params = [])
         //unregister composer
         $loaders = spl_autoload_functions();
         foreach ($loaders as $loader) {
```

Comment:
maybe it's better to name it not "$loader" to not mask function parameter with same name. My fail - can you rename it to something like "$l"?